todoapp/todos/views.py

```
@@ -1,25 +1,69 @@
+      {
+        "name": "",
+        "done": true/false,
+        "date_created": ""
+      }
+    ]
     """
 
+    permission_classes = [AllowAny]
+    serializer_class = todos_serializers.TodoDetailSerializer
+
+    def create(self, request, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for public function `create`

(ANN201)